### PR TITLE
handle request xpathes

### PR DESCRIPTION
**Pull request by MariiaNebesnova**

### CODE REVIEW

File: src/js/blocks/autoFind/autoFindProvider/pageDataHandlers.js [hunk 1]
```diff
@@ -102,12 +117,45 @@ export const runDocumentListeners = (actions) => {
...
-      .then(() => sendMessage.generateXpathes(elements, callback));
+export const requestXpathes = async (elements) => {
+  const documentResult = await connector.attachContentScript(
+    (() => JSON.stringify(document.documentElement.innerHTML))
+  );
+
+  const document = await documentResult[0].result;
+  const ids = elements.map(el => el.element_id);
+
+  const xPathResponse = await fetch("http:localhost:5000/generate_xpath", {
```
Comment (texonth) [inline src/js/blocks/autoFind/autoFindProvider/pageDataHandlers.js:128]: Just noticed that we have strange addresses. I propose to put server address in separate file or create some kind of environment checker utils. For now its ok, just a note)

**Review by texonth — APPROVED** [review]

**Comment by texonth** [comment]
Looks good for me, thanks!

File: src/js/blocks/autoFind/autoFindProvider/pageDataHandlers.js (hunk 1 above)
Comment (MariiaNebesnova) [reply src/js/blocks/autoFind/autoFindProvider/pageDataHandlers.js:128]: Good idea! I suppose to refact it with moving to remote server

**Review by Bryant-24 — APPROVED** [review]
PR looks good, thanks!

**Pull request merged**